styles.css

```diff
@@ -0,0 +1,112 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,500;0,600;0,700;1,100;1,200&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 5%;
+  height: 20%;
+  line-height: 75px;
+  background: #fff;
+  width: 95%;
+}
+
+nav .navbar-logo {
+  color: #6070ff;
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+nav .navbar-hamburger {
+  display: flex;
+  flex-direction: column;
+  height: min-content;
+  margin: 18px -10px 0 0;
+  cursor: pointer;
+}
+
+nav .navbar-hamburger .bar {
+  width: 20px;
+  height: 3px;
+  margin-bottom: 4px;
+  background: #6070ff;
+  border-radius: 18%;
+  transition: 0.4s;
+  position: relative;
+  top: 21px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  border-bottom-left-radius: 50%;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+.container {
+  background-image: url('images/shapes.svg');
+  background-size: 100% 100%;
+  width: 375px;
+  height: 580px;
+  margin-top: 10%;
+  margin-left: -3%;
+}
```

Comment:
Great job on incorporating the background image as per the previous reviewer's feedback. I really like the progress you've made with your project so far. If it's alright with you, I have a couple of suggestions that may improve your project even further:

- [ ] **[OPTIONAL]** Would it be possible to use `background-size: cover` instead of `background-size: 100% 100%`? This way, the `background image` will adjust to the size of the container, while maintaining its aspect ratio. It's just a small tweak, but it may give the project a more polished look. 👌
- [ ] **[OPTIONAL]** Additionally, you may want to consider removing the `margin-left` to correctly position the `background image`. If you still want to keep the margin for the text inside the container, you could use a [CSS child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) instead. 💯